### PR TITLE
first db schema and data setup

### PR DESCRIPTION
**Pull request by clodpheasant**

### CODE REVIEW

File: src/database/db_schema.sql [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 
-DROP TABLE IF EXISTS rates, currencies;
+DROP TABLE IF EXISTS bookmarks, users;
 
-CREATE TABLE currencies (
-  id int NOT NULL,
-  name varchar(255) NOT NULL,
-  code varchar(255) NOT NULL
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY,
```
Comment (matjack1) [inline src/database/db_schema.sql]: this could be `id` instead of `user_id`

File: src/database/db_schema.sql [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+  password VARCHAR(255)
 );
 
-CREATE TABLE rates (
-  fromcurrency_id int NOT NULL,
-  tocurrency_id int NOT NULL,
-  rate float NOT NULL,
-  timestamp timestamp NOT NULL
+CREATE TABLE bookmarks (
+  user_idd INTEGER NOT NULL,
```
Comment (matjack1) [inline src/database/db_schema.sql]: and this should be `user_id`

File: src/database/db_data.sql [hunk 3]
```diff
@@ -1,23 +1,17 @@
...
-(2, 4, 1.5, current_timestamp),
-(3, 1, 1.5, current_timestamp),
-(3, 2, 1.2, current_timestamp),
-(3, 4, 1.2, current_timestamp),
-(4, 1, 0.7, current_timestamp),
-(4, 2, 0.7, current_timestamp),
-(4, 3, 0.7, current_timestamp);
+
+INSERT INTO bookmarks (user_idd, already_known, interested_in, starred) VALUES
+(1, '{"id": "2,5,8"}','{"id": "2,5,8"}','{"id": "2,5,8"}'),
```
Comment (matjack1) [inline src/database/db_data.sql]: why instead of `'{"id": "2,5,8"}'` you are not using `'[2,5,8]'`?

File: src/database/db_schema.sql (hunk 1 above)
Comment (clodpheasant) [reply src/database/db_schema.sql]: done

File: src/database/db_schema.sql (hunk 2 above)
Comment (clodpheasant) [reply src/database/db_schema.sql]: done

File: src/database/db_data.sql (hunk 3 above)
Comment (clodpheasant) [reply src/database/db_data.sql]: modified id names

**Pull request merged**